plugins/who-can.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: who-can
+spec:
+  version: "v0.1.0-alpha.1"
+  homepage: https://github.com/aquasecurity/kubectl-who-can
+  shortDescription: |
+    Shows which users, groups and service accounts can perform a given action
+  description: |
+    Shows who has permissions to VERB [TYPE | TYPE/NAME | NONRESOURCEURL] 
+    
```

Comment:
I'd recommend to also give an example here. The description is what people see when they type `kubectl krew info who-can`, so it should be clear what exactly this plugin does. For example, you could explain that it does a reverse lookup in the rbac rules.

It would also be nice to mention that this plugin requires the rights to list (Cluster)Role and (Cluster)RoleBindings which not every user may have.